library/Data/Field.hs

```diff
@@ -178,18 +164,6 @@ instance TextField f => Show (AField f) where
     show = showField . unAField
 
 instance TextField Rational where
-    parseField = asum
-        [ try $ do
-              num <- parseFieldDefault
-              _ <- symbol "/"
-              den <- parseFieldDefault
-              case num `div` den of
-                  Just res -> pure res
-                  Nothing  -> fail $ show num ++ "/" ++ show den ++ " is not a valid Rational"
```

Comment:
I recall saying that we don't actually need to parse rationals this way as it's sufficient to parse only integer literals and defer parsing of `/` to the main parser, which recognizes that as the division operator. Not sure why that code was still there.